optimum/habana/trl/trainer/sft_trainer.py

```diff
@@ -109,6 +111,7 @@ def __init__(
         packing: Optional[bool] = False,
         formatting_func: Optional[Callable] = None,
         max_seq_length: Optional[int] = None,
+        pad_max: Optional[bool] = None,
```

Comment:
shouldnt this be set to True or False since its Bool type ?

Reply:
hi, @hsubramony there are some Bool variables that are set None too.